components/rmrk/Gallery/Gallery.vue

```diff
@@ -182,9 +180,12 @@ export default class Gallery extends mixins(PrefixMixin, InfiniteScrollMixin) {
     return this.nfts as SearchedNftsWithMeta[]
   }
 
+  get notMobile() {
+    return window.innerWidth >= 1024
+  }
```

Comment:
emm

this '1024' exists in the previous code. I just move it from `onResize` to `get notMobile`

<img width="816" alt="image" src="https://user-images.githubusercontent.com/31397967/166203998-5d218925-8617-41da-ba59-4f87646298c6.png">

Its only function is to hide searchBar input
<img width="519" alt="image" src="https://user-images.githubusercontent.com/31397967/166204638-470ee517-ce81-46e3-9864-6917b843e00b.png">

Maybe we can rename `notMobile()`with `hideSearchInput()` ?

Reply:
Okay now I understand why we have a `hideSearchInput` prop, it's legacy code that we still use in Profile section.
I thought it could be removed since we've managed to put the searchBar in the navBar.

I think you can set it to true since we'll always hideSearchInput on `/rmrk/explore/gallery`